packages/flutter/test/material/localizations_test.dart

```diff
@@ -293,4 +293,31 @@ void main() {
     expect(tester.widget<Text>(find.byKey(textKey)).data, 'id_JV');
   });
 
+  testWidgets('Localizations is compatible with ChangeNotifier.dispose() called during didChangeDependencies', (WidgetTester tester) async {
+    await tester.pumpWidget(new MaterialApp(
```

Comment:
A comment that explained the invariant we're really testing here would be heflpful.

Reply:
It was on line 305, but I moved it here since it's obviously not discoverable :)